model/post.go

```diff
@@ -80,7 +82,8 @@ type Post struct {
 	MessageSource string `json:"message_source,omitempty" db:"-"`
 
 	Type          string          `json:"type"`
-	Props         StringInterface `json:"props"`
+	propsMu       sync.RWMutex    `db:"-"`       // Unexported mutex used to guard Post.Props.
```

Comment:
Is the `db` comment needed given that this is an unexported field?

Reply:
I think so. [gorp](https://github.com/mattermost/gorp) is failing to save the post otherwise.